app/controllers/waste_exemptions_engine/last_email_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class LastEmailController < ApplicationController
+    def show
+      if ENV["ENABLE_LAST_EMAIL_CACHE"] == "true"
```

Comment:
Think I'd rather keep to the convention that config values are decided and configured by the host application rather than the engine referring to env vars directly. 

https://github.com/DEFRA/waste-exemptions-engine/blob/master/lib/waste_exemptions_engine.rb

Appreciate a bit like `USE_XVFB_FOR_WICKEDPDF` it doesn't really matter where the engine reads this value from, but for consistency I think it best to stick to the convention of host app reads the env var and then configures the engine via its config block in an initialiser.

Reply:
Fixed in 15b3b13.